hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/workload/constraints/Constraints.java

```diff
@@ -1,14 +1,33 @@
 package pl.allegro.tech.hermes.consumers.supervisor.workload.constraints;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
 public class Constraints {
 
     private final int consumersNumber;
 
-    public Constraints(int consumersNumber) {
+    @JsonCreator
+    public Constraints(@JsonProperty("consumersNumber") int consumersNumber) {
```

Comment:
probably we could move this class to `hermes-api` module

Reply:
Done.